django_elasticsearch_dsl/documents.py

```diff
@@ -91,17 +91,24 @@ def search(cls, using=None, index=None):
             model=cls.django.model
         )
 
-    def get_queryset(self):
+    def get_queryset(self, filter: models.Q = None, exclude: models.Q = None):
```

Comment:
We do not use type hint in this project.

Reply:
Ok, I removed them 👍